tests/timeline_tests/test_dashboard_sql_functions.py

```diff
@@ -367,12 +420,21 @@ def validateStartTimes(rows, rangeTypes, startTimes):
 
 def validateEndTimes(rows, rangeTypes, endTimes):
     for (row, ranget, endt) in zip_longest(rows, rangeTypes, endTimes):
+        # pylint: disable=unused-variable
         (rangetype, starttime, endtime, created_date, platid, serialid) = row
         if rangetype != ranget or endt != endtime.strftime("%H:%M:%S"):
             print(row, ranget, endt)
             return False
     return True
 
+def validateForIncludeInTimeline(rows):
+    for row in rows:
+        # pylint: disable=unused-variable
+        (recordType, includeInTimeline, serialid, pid, pname, exercise, ptname, start, end, gap, ftname, ftcolor) = row
+        if recordType == "SERIALS":
+            return includeInTimeline
```

Comment:
Is this checking exactly what you want it to be checking? As soon as it finds a value of `True` for `includeInTimeline`, it will return True, and the `assert` (used where it is called) will pass. Is that what you want?

Reply:
There is just a single row returned by the sample test data - so dropping out on the first cycle is intended.